### PR TITLE
feat: Automatically update embedded policy decision point bundles

### PR DESCRIPTION
**Pull request by haines**
This PR adds support for automatically updating embedded policy decision point bundles.

The default behaviour of the `Embedded` client is preserved - the bundle is loaded when the client is created and not updated thereafter. The actual loading is done by a new `Loader` class, and there are now optional callbacks `onLoad` and `onError` that allow the caller to handle the result of the load. The loader exposes the bundle's metadata, which was previously not available from the SDK.

The new behaviour is implemented in a subclass of the default `Loader`, `AutoUpdatingLoader`, which polls the bundle URL on a configurable interval (defaulting to 1 minute). By default, updates take effect immediately; by setting `activateOnLoad` to `false`, they may instead be activated manually using the loader's `activate` method. This caters for the use case where you only want to apply bundle updates on page transitions to avoid layout shifts from policy changes.

### CODE REVIEW

File: packages/embedded/src/loader.ts [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+    private readonly url: string | URL,
+    {
+      activateOnLoad = true,
+      interval = 60000,
+      ...options
+    }: AutoUpdateOptions = {},
+  ) {
+    super(url, options);
+    this.activateOnLoad = activateOnLoad;
+    this.interval = interval;
```
Comment (charithe) [inline packages/embedded/src/loader.ts:344]: Should we cap the minimum interval value to prevent users from providing a really small value?

Comment (haines) [reply packages/embedded/src/loader.ts:344]: Yep, seems reasonable. What do you think should be the minimum? Maybe 10 seconds?

Comment (charithe) [reply packages/embedded/src/loader.ts:344]: Yeah, 10 seconds seems reasonable. I can't think of an application that would require more frequent updates.

**Review by charithe — APPROVED** [review]
🙌🏽

**Pull request merged**